canonical/ubuntu.com
# [WD-16570] Redirect - from /download/RPi to /core/docs

## Done

- Redirect /download/raspberry-pi-core to /core/docs/install-with-imager

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Make sure that http://0.0.0.0:8001/download/raspberry-pi-core redirects to /core/docs/install-with-imager

## Issue / Card

Fixes WD-16570

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
